proteuslib/unit_models/reverse_osmosis_0D.py

```diff
@@ -654,12 +709,9 @@ def eq_concentration_polarization_io(b, t, x, j):
                         prop_io.conc_mass_phase_comp['Liq', j]
                         * self.cp_modulus[t, j])
             elif self.config.concentration_polarization_type == ConcentrationPolarizationType.calculated:
-                jw = self.flux_mass_io_phase_comp[t, x, 'Liq', 'H2O'] / self.dens_solvent
+                jw = self.flux_mass_io_phase_comp[t, x, 'Liq', self.solvent_list[1]] / self.dens_solvent
```

Comment:
I think I had originally changed this to `self.solvent_list[1]` here like you did. 

Then, @andrewlee94 recommended the use of `self.solvent_list.first()` and then upon further reflection suggested using `H2O` directly here since RO is only setup for that. A small detail, but all that just to suggest that we should keep what was there.